examples/configurations/private-link-schema-registry/main.tf

```diff
@@ -7,7 +7,7 @@ terraform {
     }
     confluent = {
       source  = "confluentinc/confluent"
-      version = "1.81.0"
+      version = "2.16.0"
```

Comment:
same thing as https://github.com/confluentinc/terraform-provider-confluent/pull/549/files#r1943631786

Reply:
I went ahead and patched the example 🫡